File: AndroidArtStudy/Chapter13/chapter13_1/src/main/java/com/example/chapter13_1/MyApplication.java
package com.example.chapter13_1;

import android.app.Application;

/**
 * Created by liu on 2016/12/15 0015.
 */

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        CrashHandler crashHandler = CrashHandler.getInstance();
        crashHandler.init(this);
    }
}

File: AndroidArtStudy/Chapter02/demo01/src/main/java/com/example/demo01/MyApplication.kt
package com.example.demo01

import android.app.Application
import android.os.Process
import android.util.Log

/**
 * Created by liu on 2016/11/24 0024.
 */
class MyApplication : Application() {
    companion object {
        val TAG = "MyApplication"
    }

    override fun onCreate() {
        super.onCreate()
        val processName = getProcessName(applicationContext, Process.myPid())
        Log.d(TAG, "application start, process name: " + processName)
    }
}